runDocker.ts

```diff
@@ -0,0 +1,94 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import Dockerode, { Container } from 'dockerode'
+import { fork } from 'node:child_process'
+import { LauncherFunction } from './run.js'
+
+const [, , command, jsonifiedArgs] = process.argv
+const docker = new Dockerode({ protocol: 'http' })
+
+let containerId = ''
+if (command === 'launch-ddb-local-docker-subprocess') {
+  const { port } = JSON.parse(jsonifiedArgs)
+  let container: Container
+  try {
+    container = await createDdbContainer(port)
+  } catch (error) {
+    // Fix for Windows, containers exit, but do not get removed properly.
+    console.log('\nExisting container, removing and recreating')
+    const containers = await docker.listContainers({
+      limit: 1,
+      filters: '{"name": ["dynamodb-local"]}',
```

Comment:
This will remove _any_ container that happens to have this name. Can you give it a unique name and store it, so that you don't accidentally kill other containers that happen to have the same name?

Reply:
You can't have docker containers with the same name, this shouldn't be an issue.